desktop/eventListners.js

```diff
@@ -84,6 +85,14 @@ const subscribeToEventListeners = ({ mainWindow }) => {
   ipcMain.on(ipcConsts.SET_NODE_IP, async (event, request) => {
     netService.setNodeIpAddress({ event, ...request });
   });
+
+  ipcMain.on(ipcConsts.NOTIFICATION_CLICK, async (event) => {
```

Comment:
func calls of show and focus are synchronous so no need to declare async for callback
also the notificationsManager is excess class, the response can be send right in this event listener

Reply:
np